### PR TITLE
Added "Internet Requests are Broken on Android" article

### PR DESCRIPTION
**Pull request by PreslavKozovski**

### CODE REVIEW

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Hybrid application that is deployed on an Android device cannot do any internet requests.
+
+## Error Message
+
+Failed to load resource: the server responded with a status of 404 (Not Found) 
+
+## Cause
+
+{{site.ac}} 4.0 and above incorporates the {{site.ac}} whitelisting system, which provides a configurable security policy. For android, this policy is implemented by the [{{site.ac}} Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/). If the plugin is not enabled in the project, any internet request will respond with a status of 404 (Not Found).
```
Comment (PreslavKozovski) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: Android

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Solution
+
+To enable the {{site.ac}} Whitelist Plugin and allow the internet request, complete the following steps:
+
+1. Open project **Properties**.
+1. Navigate to **Plugins** tab.
+1. Navigate to the last one of the **Core Plugins** section, which is **Whitelist**
+1. If the plugin switch is **OFF**, click on it to enable the plugin for the project.
+1. **Rebuild** and **deploy** the project on the android device.
```
Comment (PreslavKozovski) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: Android

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  </tr>
+  <tr>
+    <td>{{site.ac}} framework version</td>
+    <td>{{site.ac}} 4.0.0 and later</td>	
+  </tr>
+</table>
+
+## Description
+
+Hybrid application that is deployed on an Android device cannot do any internet requests.
```
Comment (AntonDobrev) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: internet - HTTP/S requests

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+{{site.ac}} 4.0 and above incorporates the {{site.ac}} whitelisting system, which provides a configurable security policy. For android, this policy is implemented by the [{{site.ac}} Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/). If the plugin is not enabled in the project, any internet request will respond with a status of 404 (Not Found).
+
+## Solution
+
+To enable the {{site.ac}} Whitelist Plugin and allow the internet request, complete the following steps:
+
+1. Open project **Properties**.
+1. Navigate to **Plugins** tab.
+1. Navigate to the last one of the **Core Plugins** section, which is **Whitelist**
+1. If the plugin switch is **OFF**, click on it to enable the plugin for the project.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: Mist/Proton Core Plugins do not have ON/OFF switch. You might consider using enabled/disabled.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+title: jQuery/AJAX/JSON Requests do not Work on Android
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: I don't believe this title is descriptive and I don't like the slashes. Suggestion: 
Ajax jQuery Requests Do Not Work on Android
or
Ajax jQuery Requests Are Not Working on Android

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+description: jQuery API does not return any data, never fires on a real Android device.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: Never fires or does not return any data? I don't think it's the same. 
You can also put the JSON word here if you think it is needed. However, I believe this should happen if only JSON files are not returned, which I think is not the case. 
For example:
On Android devices, requesting JSON through jQuery AJAX does not return any data.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+type: troubleshooting
+page_title: Internet Requests are Broken on Android.
```
Comment (mkapostolov) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: I think it is better to say: All HTTP/HTTPS requests return "404 (Not Found)" on Android
Internet Requests are Broken on Android - sounds like bad English translation.

Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: 👍

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md (hunk 1 above)
Comment (mkapostolov) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: incorporates the {{site.ac}} domain whitelisting, a security model which provides a configurable security policy to define which external sites may be accessed.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md (hunk 3 above)
Comment (ggarabedian) [reply troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: You can substitute **do** with make/perform/execute.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md (hunk 1 above)
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: As Anton suggested above -> internet - HTTP/S requests. Internet requests sounds too vague.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+## Cause
+
+{{site.ac}} 4.0 and above incorporates the {{site.ac}} domain whitelisting, a security model which provides a configurable security policy to define which external sites may be accessed. For Android, this policy is implemented by the [{{site.ac}} Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/). If the plugin is not enabled in the project, any HTTP/HTTP request will respond with a status of 404 (Not Found).
+
+## Solution
+
+To enable the {{site.ac}} Whitelist Plugin and allow the HTTP/HTTP requests, complete the following steps:
+
+1. Open project **Properties**.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md:55]: Personal preference. Usually, the first step is: Open your project in AppBuilder
Then:
From the Project Navigator (Solution Explorer in Microsoft Visual Studio), select Properties.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+1. Navigate to **Plugins** tab.
+1. Navigate to the last one of the **Core Plugins** section, which is **Whitelist**
+1. If the plugin is **disabled**, click on it to **enable** the plugin for the project.
+1. **Rebuild** and **deploy** the project on the Android device.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: **Build** and...

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+## Cause
+
+{{site.ac}} 4.0 and above incorporates the {{site.ac}} domain whitelisting, a security model which provides a configurable security policy to define which external sites may be accessed. For Android, this policy is implemented by the [{{site.ac}} Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/). If the plugin is not enabled in the project, any HTTP/HTTP request will respond with a status of 404 (Not Found).
+
+## Solution
+
+To enable the {{site.ac}} Whitelist Plugin and allow the HTTP/HTTP requests, complete the following steps:
+
+1. Open project **Properties**.
+1. Navigate to **Plugins** tab.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: Open the **Plugins** tab.

File: troubleshooting/troubleshooting-android-whitelist-not-enabled.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+1. Navigate to the last one of the **Core Plugins** section, which is **Whitelist**
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-android-whitelist-not-enabled.md]: In the **Core Plugins** section, locate the **Whitelist** plugin.

**Comment by AntonDobrev** [comment]
👍

**Comment by ggarabedian** [comment]
👍

**Pull request merged**